### PR TITLE
Add home page

### LINKED ISSUE #5: add hompage non animated components
Create homepage non animated components

### PR DESCRIPTION
**Pull request by usq2**
- Added all non animated home page components

### CODE REVIEW

File: src/component/ui_helpers/ProfileButton.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      color: '#fff',
+    },
+  });
+  return (
+    <View style={styles.profileButton}>
+      <Pressable
+        onPress={() => {
+          navigation.navigate('Profile');
+        }}>
+        <Text style={styles.profileButtonText}>D</Text>
```
Comment (dinchen1089) [inline src/component/ui_helpers/ProfileButton.jsx]: D can be passed as a parameter

File: src/component_data/secondaryActionsData.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      console.log('kettlebell pressed');
+    },
+  },
+  {
+    bottomText: 'Apparel',
+    key: 'cult-secondary-3',
+    Icon: <TShirtIcon color={'#fff'} size={40} />,
+    style: {backgroundColor: 'rgba(211,211,211,0.2)', paddingVertical: 30},
+    onPress: navigation => {
+      console.log('weight lifter pressed');
```
Comment (dinchen1089) [inline src/component_data/secondaryActionsData.js]: Remove this

File: src/component_data/secondaryActionsData.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    },
+  },
+  {
+    bottomText: 'Dumbbell',
+    key: 'cult-secondary-4',
+    Icon: <DumbbellIcon color={'#fff'} size={40} />,
+    style: {backgroundColor: 'rgba(211,211,211,0.2)', paddingVertical: 30},
+    onPress: navigation => {
+      console.log('buy sports wear pressed');
```
Comment (dinchen1089) [inline src/component_data/secondaryActionsData.js]: Remove this or add todo if required

File: src/images/imageMap.js [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+      },
+    },
+  },
+};
+export const HomeExploreSectionMap = {
+  exploreRunner: {
+    path: require('../images/runner.jpg'),
+    id: 'home-explore-2',
+    bottomText:
+      'Make stubborn fat disappear with sustainable weight loss coaching',
```
Comment (dinchen1089) [inline src/images/imageMap.js:40]: Add this text in english if this text wont change

File: src/screen/Home.js [hunk 5]
```diff
@@ -1,19 +1,181 @@
...
+        <Text
+          style={{
+            fontSize: 14,
+            color: '#a9a9a9',
+            textAlign: 'center',
+            width: '70%',
+            marginTop: 10,
+            textAlignVertical: 'center',
+          }}>
+          Everything you need to stay inspired & stay on top of your game
```
Comment (dinchen1089) [inline src/screen/Home.js]: Add his text in english file

**Review by dinchen1089 — CHANGES_REQUESTED** [review]

File: src/component_data/secondaryActionsData.js (hunk 2 above)
Comment (usq2) [reply src/component_data/secondaryActionsData.js:43]: no i will not

**Pull request merged**